tutorials/logical_audio_sensor.md

```diff
@@ -0,0 +1,195 @@
+  }
+}
+data: 0.4900980486407216
+```
+
+This means that this microphone can detect the source connected to `red_box`, and the volume level detected by the microphone was roughly `0.49` (0.0 being 0% volume, and 1.0 being 100% volume).
+Now, go ahead and start simulation by pressing the "play" icon at the bottom-left of the screen, and the detection output will now have a time stamp attached to it:
+
+```
+header {
+  stamp {
+    sec: 8
+    nsec: 925000000
+  }
+  data {
+    key: "world/logical_audio_sensor/model/red_box/sensor/source_1"
+  }
+}
+data: 0.4900980486407216
+```
+
+Since the source attached to `red_box` was defined in the SDF file using `<play_duration>10</play_duration>`, this means that the last message published by the `green_box` detection topic will have a time stamp of 10 seconds.
+
+### Starting/Stopping Audio Sources
+
+Now that we've discussed how to observe microphone detections, let's see how we can modify the state of a logical audio source.
+Logical audio sources can be started/stopped manually through Ignition services.
+To see which services to call in order to start/stop a service, open a new terminal and run the following command (make sure the simulator is still running):
+
+```
+ign service -l
+```
+
+If you look through the list of available services, you should see the following audio source services (the `_1` suffix is the ID assigned to the sources in the SDF):
+
+```
+/model/blue_box/sensor/source_1/play
+/model/blue_box/sensor/source_1/stop
+/model/red_box/sensor/source_1/play
+/model/red_box/sensor/source_1/stop
+```
+
+Let's start the source attached to `blue_box`.
+This can be done by running the following command, which calls the "play" service for the source attached to `blue_box`:
+
+```
+ign service -s /model/blue_box/sensor/source_1/play --reqtype ignition.msgs.Empty --reptype ignition.msgs.Boolean --timeout 1000 --req 'unused: false'
+```
+
+Now, if you look back at the terminal that is displaying the output of `green_box`'s microphone detections, we can see that this microphone is detecting audio from `blue_box`'s source (we see `blue_box` as a part of the `key` field):
+
+```
+header {
+  stamp {
+    sec: 255
+    nsec: 673000000
+  }
+  data {
+    key: "world/logical_audio_sensor/model/blue_box/sensor/source_1"
+  }
+}
+data: 0.6
+```
+
+You can also echo `yellow_box`'s microphone detection topic, and you should see that this microphone can detect audio from `blue_box`'s source as well.
+
+Since the source attached to `blue_box` was configured with `<play_duration>0</play_duration>` in SDF, it won't stop playing unless we call the stop service on it.
+Before we call the stop service on this source, move `blue_box` around with the transform control tool - you'll see that once `blue_box`'s source volume falls below a microphone's detection threshold (defined in the SDF via `<volume_threshold>`), messages will stop being published to the microphone's detection topic.
```

Comment:
```suggestion
Before we call the stop service on this source, move `blue_box` around with the transform control tool. You'll see that once `blue_box`'s source volume falls below a microphone's detection threshold (defined in the SDF via `<volume_threshold>`), messages will stop being published to the microphone's detection topic.
```

Reply:
Fixed in https://github.com/ignitionrobotics/ign-gazebo/pull/401/commits/bb50ab5e21d6fec3c54a1e1b7fdd6623fc285908